python/sklearnserver/sklearnserver/model.py

```diff
@@ -0,0 +1,33 @@
+import kfserving
+from joblib import load
+import numpy as np
+import os
+
+
+class SKLearnModel(kfserving.KFModel):
+    def __init__(self, name, model_dir):
+        self.name = name
+        self.model_dir = model_dir
+        self.ready = False
+
+    def load(self):
+        model_file = kfserving.Storage.download(self.model_dir)
+        self._model = load(model_file)
+        self.ready = True
+
+    def preprocess(self, inputs):
```

Comment:
This pre/post/predict pattern is getting refactored soon. Can you put it all into the predict function?

Reply:
My suggestion would be refactor all at once when we get to it.

Reply:
Done - left the pre and post as dummy methods for now.

Reply:
If you leave these out completely, it should still wire up as the parent impl is the same as this.

Reply:
Done